package.json

```diff
@@ -76,16 +77,15 @@
     "@expo/spawn-async": "^1.5.0",
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-node-resolve": "^7.1.1",
-    "@storybook/addon-a11y": "^5.2.6",
-    "@storybook/addon-actions": "^5.2.3",
-    "@storybook/addon-info": "^5.2.3",
-    "@storybook/addon-knobs": "^5.2.3",
-    "@storybook/addon-links": "^5.2.3",
-    "@storybook/addon-storyshots": "^5.2.3",
-    "@storybook/addon-viewport": "^5.3.14",
-    "@storybook/addons": "^5.2.3",
-    "@storybook/core": "^5.3.14",
-    "@storybook/react": "^5.2.3",
+    "@storybook/addon-a11y": "^6.1.10",
+    "@storybook/addon-actions": "^6.1.10",
+    "@storybook/addon-links": "^6.1.10",
+    "@storybook/addon-essentials": "^6.1.10",
+    "@storybook/addon-storyshots": "^6.1.10",
+    "@storybook/addon-viewport": "^6.1.10",
+    "@storybook/addons": "^6.1.10",
+    "@storybook/core": "^6.1.10",
+    "@storybook/react": "^6.1.10",
```

Comment:
Just throwing this here because I couldn't comment where the linter was hollering at the knobs imports.

https://storybook.js.org/blog/storybook-controls/

Controls replaces knobs the api is nicer and pretty auto-magic which is really helpful.